lib/wiki_discouraged_article_manager.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require_dependency "#{Rails.root}/lib/importers/category_importer"
+
+class WikiDiscouragedArticleManager
+  def retrieve_wiki_edu_discouraged_articles
+    category_titles = [ENV['blocked_assignment_category'],
+                       ENV['warning_assignment_category']]
```

Comment:
Again, I think this will be better if we cut out the `warning` feature for this PR and just do the blocked assignments as an initial implementation. (That will make it much easier for me to review, easier to write tests for, and that's the feature I know that we will want to use immediately.)

Reply:
Okay, i will remove the `warning` feature.